.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!-- Feel free to remove check-list items aren't relevant to your change -->
 
-- [ ] Closes #xxxx
-- [ ] Tests added
-- [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
-- [ ] New functions/methods are listed in `api.rst`
+- \[ \] Closes #xxxx
```

Comment:
Are we confident this still displays correctly?

Reply:
It seems to be [fine](https://github.com/pydata/xarray/blob/097e54955da321b965d00b9fc00d391433680793/.github/PULL_REQUEST_TEMPLATE.md), though I also find that surprising.

Reply:
Indeed!

Though it seems a bit confusing since people will be directly editing it...

Reply:
Ok, then I'll ignore this file.